updatecli-compose.yaml

```diff
@@ -0,0 +1,8 @@
+# Config file for `updatecli compose ...`.
+# https://www.updatecli.io/docs/core/compose/
+policies:
+  - name: Local Updatecli Policies
+    config:
+      - .github/updatecli.yml
+    values:
+      - .github/values.d/scm.yml
```

Comment:
I prefer we use `updatecli/updatecli.d/` for config and `updatecli/values.d/scm.yml` for values.